ptimeit/_impl.py

```diff
@@ -34,18 +42,31 @@ def timeit_section(section_name: str):
     finally:
         elapsed_time_s = time.monotonic() - start_time
         elapsed_time_ms = round(float(elapsed_time_s * 1000), 1)
-        elapsed_time_ms_str = str(elapsed_time_ms).rjust(14)
-        section_full_name = '|   ' * _depth + section_name
-        print(f'->>>>>>>>{elapsed_time_ms_str}ms      {section_full_name}')
+        if condition is None or condition(elapsed_time_ms):
+            elapsed_time_ms_str = str(elapsed_time_ms).rjust(14)
+            offset = '|   ' * _depth
+            section_full_name = offset + section_name
+            to_print=f'->>>>>>>>{elapsed_time_ms_str}ms      {section_full_name}'
+            if additional_data_to_print:
+                to_print += f'- {additional_data_to_print}'
```

Comment:
Hmm, feels like we need an extra space here or it
's handled somewhere else ? 🤔 
```suggestion
                to_print += f' - {additional_data_to_print}'
```

Reply:
https://github.com/narekgharibyan/ptimeit/pull/1/commits/d3ff11903ec596d04d4eafe52b02d500a48bf187